dirty_cat/test/test_minhash_encoder.py

```diff
@@ -40,6 +40,17 @@ def test_MinHashEncoder(n_sample=70, minmax_hash=False):
                 y_substring = encoder.transform(X_substring)
                 np.testing.assert_array_less(y - y_substring, 0.0001)
 
+def test_multiple_columns():
+    """ This test is intented to verify that fitting multiple columns
+        with the MinHashEncoder will not produce an error, but will 
+        encode the column independently """
+    X = pd.DataFrame([('bird', 'parrot'),
+                      ('bird', 'nightingale'),
+                      ('mammal', 'monkey'),
+                      ('mammal', np.nan)],
+                      columns=('class', 'type'))
+    MinHashEncoder().fit_transform(X)
```

Comment:
We should push a bit this test. For instance we could compare this to column-wise encoding.